### ISSUE TITLE
Problems in debug build: Cannot deserialize XBF metadata type list as 'BootStrapper' 

### ISSUE THREAD

**Issue by WieserSoftware** [Windows-XAML/Template10#1291]
I've installed template 10, version 1.18
If I create a new hamburger project, and try to run it in debug mode on my pc, I get an exception on line 53 of app.g.i.cs:

```
# if DEBUG && !DISABLE_XAML_GENERATED_BREAK_ON_UNHANDLED_EXCEPTION
        UnhandledException += (sender, e) =>
        {
            if (global::System.Diagnostics.Debugger.IsAttached) global::System.Diagnostics.Debugger.Break();
        };
# endif
```

The exception is: 
> Cannot deserialize XBF metadata type list as 'BootStrapper' was not found in namespace 'Template10.Common'. [Line: 0 Position: 0]

When I created the project, I chose target version : Windows 10 anniversary edition (10.0; 14393)
and Minimum version: Windows 10 (10.0; build 10586)

If I select a release build instead of debug, everything works fine.

I'm using VS2015 enterprise, 14.0.25431.01 Update 3
I have updated to the latest windows updates, Version 1607, build 14393.351

**Comment by WieserSoftware** [comment]
I also notice I have some warnings:
Severity    Code    Description Project File    Line    Suppression State
Warning     Detected package downgrade: Microsoft.NETCore.UniversalWindowsPlatform from 5.2.2 to 5.1.0 
 GitHamburger2 (>= 1.0.0) -> Template10 (>= 1.1.0) -> Microsoft.NETCore.UniversalWindowsPlatform (>= 5.2.2) 
 GitHamburger2 (>= 1.0.0) -> Microsoft.NETCore.UniversalWindowsPlatform (>= 5.1.0)          0

**Comment by WieserSoftware** [comment]
It appears that the problem is caused by the template defaulting to  
"Template10": "1.1.*"
in project.json
Alternatively, this line should be updated to 5.2.2
    "Microsoft.NETCore.UniversalWindowsPlatform": "5.1.0",

It appears that this issue was closed as #1177 and changes were made on 22/9 but still occurs on latest install of the extension
